src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/classes/eltwise.cpp

```diff
@@ -292,7 +292,8 @@ std::string EltwiseLayerCPUTest::getPrimitiveType(const utils::EltwiseTypes& elt
         if ((eltwise_type == utils::EltwiseTypes::ADD) ||
             (eltwise_type == utils::EltwiseTypes::SUBTRACT) ||
             (eltwise_type == utils::EltwiseTypes::MULTIPLY) ||
-            (eltwise_type == utils::EltwiseTypes::DIVIDE)) {
+            (eltwise_type == utils::EltwiseTypes::DIVIDE) ||
+            (eltwise_type == utils::EltwiseTypes::BITWISE_XOR)) {
```

Comment:
As I mentioned in the previous comment, the target operation is `LogicalXor`, not `BitwiseXor`.

`logical` ops are tested [there](https://github.com/openvinotoolkit/openvino/blob/58b02fb88fa660468d281e751f117c51cb22932c/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/logical.cpp#L76). You don't need to update any checks for primitive type (`getPrimitiveType`) there as for eltwise/activation ops.

So please just check that the tests with `--gtest_filter="*smoke*Logical*LogicalXor*"` are successfully passed.